osu.Game/Screens/OnlinePlay/Multiplayer/IMultiplayerMatchScreen.cs

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Screens;
+using osu.Game.Online.Rooms;
+
+namespace osu.Game.Screens.OnlinePlay.Multiplayer
+{
+    public interface IMultiplayerMatchScreen
+    {
+        Room Room { get; }
+
+        bool IsCurrentScreen();
+
+        void Push(IScreen screen);
```

Comment:
these two existing here is highly weird, would expect them to be in `IScreen` and then `IMultiplayerMatchScreen : IScreen`